code/modules/research/designs.dm

```diff
@@ -175,7 +175,7 @@ datum/design/proc/CalcReliability(var/list/temp_techs)
 	id = "jackhammer"
 	req_tech = list("materials" = 6, "powerstorage" = 6, "engineering" = 5, "magnets" = 6)
 	build_type = PROTOLATHE
-	materials = list("$metal" = 8000, "$glass" = 1500, "$silver" = 2000, "$diamond" = 6000)
+	materials = list("$metal" = 8000, "$glass" = 1500, "$silver" = 2000, "$diamond" = 3750)

```

Comment:
The jackhammer is possibly more expensive since it still requires more diamond than the adv plasmacutter. Also you could make 2 of the cutters for this amount you put in if you only got 1 gold silver and diamond. Your fix appears to only add half a diamond and almost half a jackhammer. Should it be made the plasma cutter can only be brought at the vendor for a lot of points?